XamlControlsGallery/ControlPages/ProgressRingPage.xaml

```diff
@@ -20,15 +20,12 @@
     mc:Ignorable="d">
     <StackPanel>
         <local:ControlExample HeaderText="An indeterminate progress ring.">
-
-            <muxc:ProgressRing IsActive="{x:Bind ProgressToggle.IsOn, Mode=OneWay}" Height="60" Width="60" VerticalAlignment="Top" Margin="10,10,0,0"/>
-
+            <muxc:ProgressRing IsActive="{x:Bind ProgressToggle.IsOn, Mode=OneWay}" Height="130" Width="130" VerticalAlignment="Top" Margin="10,10,0,0"/>
```

Comment:
Height and width of 130 looks a bit out of place in my opinion, maybe we could choose a smaller size?

Reply:
I tend to agree. This feels a little big, and I'd like to see us stay on 4px grid. So 120 would already be a better size from that perspective.

Reply:
ok no problem I can switch it back to 60.